README.md

```diff
@@ -127,12 +128,11 @@ See [DEVELOPMENT.md](./DEVELOPMENT.md)
 
 2. `Whisper.transcribe(arr: NDArray[np.float32], num_proc: int = 1)`
 
-   Running transcription on a given Numpy array. This calls `full` from
-   `whisper.cpp`. If `num_proc` is greater than 1, it will use `full_parallel`
-   instead.
+   Running transcription on a given Numpy array. This calls `full_parallel` from
+   `whisper.cpp`.
```

Comment:
what is the motivation behind this change? it calls `full` when num_proc is 1

Reply:
Yeah, my bad. Reverted.